helpers/testhelpers/testhelpers.go

```diff
@@ -286,10 +287,39 @@ func PrepareExternalServerCall(t *testing.T, c *controllers.SecureContext, testS
 	return nil, nil, nil
 }
 
+// comparePossiblyJSONStrings will try to parse both as JSON. If it succeeds, it will marshal them both out and compare.
+// Since JSON doesn't actually have a canonical form, this may be flaky, but likely less flaky than previous behavior.
+// If anything fails, it will compare the original string as strings
+func comparePossiblyJSONStrings(a, b string) bool {
+	fallback := (a == b)
+
+	var oa, ob interface{}
```

Comment:
we can use testify's `JSONEq` https://github.com/stretchr/testify/blob/890a5c3458b43e6104ff5da8dfa139d013d77544/assert/assertions.go#L1128